### ISSUE TITLE
Enhancement : Force Directed Map Layouts

### ISSUE THREAD

**Issue by IvandaNothabeer** [exodus4d/pathfinder#201]
Force Directed maps are a (relatively) simple way to provide an automatic layout for maps that avoid overlaps etc.

Something like getspringy ...  http://getspringy.com/

**Comment by IvandaNothabeer** [comment]
Seems that jsPlumb implements a Force Directed layout as well ... https://jsplumbtoolkit.com/docs/toolkit/demo-spring.html#adding

**Comment by exodus4d** [comment]
Yeah this is from the "Tookit Edition" which costs about $2500,.... https://jsplumbtoolkit.com/download
I use the "Community Edition" (free) with some custom plugins. 

We could use the "magnetize" feature for this (with some modifications). I have implemented this  for testing (some month ago) and can be activated on the "maps menu".
This is not a "top priority" feature request, more a "nice to have" feature....

**Comment by dagoaty** [comment]
Can I add a vote for this. Coming from another mapping tool (eve-wspace) it is surprising how being able to freely place the systems adds confusion. Turning on magnetising helps a bit with the initial placement but it would be nice to force a rows and columns system in some way.

**Comment by w9jds** [comment]
I wrote an algorithm for jsplumb that forces a treeview layout if you would like to modify it, let me know and I can throw it into a gist. It isn't perfect, but it is pretty decent for what it does. Figured it was WAY better than paying that gouging 2.5k price per year to use their stupid toolkit.
